viewPost.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+include('config/init.php');
+
+
+$result=getAllBlogPosts();
+$blogPosts= getBlogPost($_GET['categoryId']);
+
+
+echoHeader("".$blogPosts['title'], "".$blogPosts ['description']);
+
+echo "
+<h2>".$blogPosts['title']."</h2>
+<p>".$blogPosts['content']."</p> 
+";
+
+if(isset($_REQUEST['newComment'])){
+    createComment(
+        $_REQUEST['content']
+    );
+}
+?>
+ 
+ 
+ 
+ <h3>Comments</h3>
+<form action='' method='post'>
+    Comment:
+    <input type='text' name='content'/><br/>
+    <input type='submit' name='newComment' value='Post'/>
```

Comment:
Have you considered taking someone's name/email here. It's fine if you're just trying to get something basic for now, but eventually you might get spammers or something like that, and having more fields can help you filter spam out.